source/server/config/network/original_dst.cc

```diff
@@ -0,0 +1,47 @@
+#include <string>
```

Comment:
Please move the configs to a listener folder, like the filters.

Reply:
Will do.